docs/community/com-query/grafana-connector.md

```diff
@@ -51,26 +58,26 @@ From `grafana-7-x.x/` run the following command:
 ### Add Datasource
```

Comment:
This should be a completely new page because it's a different task than the installation.

Or put it another way, does the user need to install **everything** each time they want to add a datasource? 

I suspect not. Installation and setup is performed once. Adding datasources can happen at any time.

Reply:
@rachithrr you can skip this